<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemoveLoan</class>
 <widget class="QMainWindow" name="RemoveLoan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>471</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="horizontalSpacing">
      <number>50</number>
     </property>
     <property name="verticalSpacing">
      <number>20</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
       </property>
       <property name="text">
        <string>UserID</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="in_userID">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="in_itemID">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
       </property>
       <property name="text">
        <string>ItemID</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="LoanDetails" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>681</width>
      <height>331</height>
     </rect>
    </property>
    <widget class="QWidget" name="formLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>471</width>
       <height>141</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="horizontalSpacing">
       <number>23</number>
      </property>
      <property name="verticalSpacing">
       <number>21</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
        </property>
        <property name="text">
         <string>Loan ID</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="out_loanID">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
        </property>
        <property name="text">
         <string>Issue Date</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="iDate">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>170</y>
       <width>261</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>29</number>
      </property>
      <item>
       <widget class="QPushButton" name="renew_button">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
        </property>
        <property name="text">
         <string>Renew Item</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="check_in_button">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
        </property>
        <property name="text">
         <string>Check In Item</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="search_button">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>180</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>Search Loan</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
